packages/shared/lib/core/network/classes/iscp-chain.class.ts

```diff
@@ -31,16 +33,38 @@ export class IscpChain implements IChain {
         return `v1/chains/${aliasAddress}/evm`
     }
 
-    async getLatestBlock(): Promise<IBlock> {
-        const number = await this._provider.eth.getBlockNumber()
-        return this._provider.eth.getBlock(number)
+    getConfiguration(): ChainConfiguration {
+        return this._configuration
+    }
+
+    getMetadata(): Promise<ChainMetadata> {
+        if (this._metadata) {
+            return this._metadata
+        } else {
+            this._metadata = <IIscpChainMetadata>{} // await this.fetchChainMetadata()
+            return this._metadata
+        }
```

Comment:
not sure if this is still needed?

Reply:
I think we should keep for the ISCP chain metadata we can fetch later